modal/image.py

```diff
@@ -644,7 +644,7 @@ def poetry_install_from_file(
     @typechecked
     def dockerfile_commands(
         self,
-        dockerfile_commands: Union[str, List[str]],
+        *dockerfile_commands: Union[str, List[str]],
         context_files: Dict[str, str] = {},
```

Comment:
i guess this could break old code that passes `context_files` as a positional argument. maybe fine?

Reply:
Good idea to add `*`. This method isn't super widely used so I think it's fine to jump straight to `*args`